packages/code/src/main.rs

```diff
@@ -94,13 +94,113 @@ fn transform_code(to: &str) {
                 print!("[{}]", json!({"name": "raw", "data": html}));
             }
         }
+        "latex" => {
+            if font_size != 12{
+                eprintln!("Font size is not supported in LaTeX");
+            }
+            print!("{}", highlight_latex(code, lang, theme, tab_size));
+        }
         other => {
             eprintln!("Cannot convert code to {other}");
         }
     }
 }
 
-fn get_style(
+fn highlight_latex(code: &str, lang: &String, tm: &str, tab_size: u64) -> String {
+    let ss = SyntaxSet::load_defaults_newlines();
+    let ts = ThemeSet::load_defaults();
+    let theme = match tm {
```

Comment:
We have two (to me) identical theme matches, and loading of `SyntaxSet` and `ThemeSet`, might be nicer to not do that in two places